code/modules/projectiles/projectile/reusable/foam_dart.dm

```diff
@@ -20,8 +20,9 @@
 	newcasing.modified = modified
 	var/obj/projectile/bullet/reusable/foam_dart/newdart = newcasing.BB
 	newdart.modified = modified
-	newdart.damage = damage
-	newdart.nodamage = nodamage
+	if(modified)
+		newdart.damage = 5
```

Comment:
lowkey, I wouldn't mind if this was buffed to be like 10 damage

5 damage from a projectile that you have to specially prepare (with a hard to mass-produce resource, pens) is GARBAGE, even when burst fired

Reply:
not my problem this is an exploit fix, make the pr after please

Reply:
It's not supposed to be a pro gamer move, it's supposed to be slightly malicious and mildly obnoxious.